wandb/wandb_config.py

```diff
@@ -217,6 +238,7 @@ def _sanitize_val(self, val):
             return val
 
     def _update(self, params, allow_val_change=False, as_defaults=False):
```

Comment:
Do we want params=None here?

Reply:
None might get passed from wandb.init(), before we were checking `if config:` but now we want to update telemetry also